README.md

```diff
@@ -1,13 +1,18 @@
+Tone platform
+=============
 Copyright (C) Sony Mobile Communications 2016
-=============================================
+---------------------------------------------
 
-This is the Android device configuration for tone platform.
+Android device configuration for the tone platform (**MSM8996**).
 
-tone devices:
+### Supported devices
 
-    Xperia X Performance ................ Dora
-    Xperia XZ ........................... Kagura
-    Xperia XZs .......................... Keyaki
+| Device | Codename |
+|-|:-:|
+| Xperia X Performance | [Dora](https://github.com/sonyxperiadev/device-sony-dora) |
```

Comment:
Strangely you made kagura and keyaki lowercase, but left dora uppercase?

Reply:
Ah mistake on my side. I will fix it up today.